src/test/util/matchable.rs

```diff
@@ -67,9 +67,12 @@ impl Matchable for Document {
             if k == "afterClusterTime" {
                 continue;
             }
-            // TODO RUST-97: Remove this logic to bypass recoveryToken
-            if k == "recoveryToken" {
-                continue;
+            if k == "recoveryToken" && v.is_placeholder() {
```

Comment:
[`recoveryToken` is tested as follows.](https://github.com/mongodb/specifications/blob/master/source/transactions/tests/README.rst#recoverytoken)